### PR TITLE
refactor(app): use create-react-app for client & merge electron + server

### PR DESCRIPTION
**Pull request by JPSchellenberg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +92,7 @@
         "@sentry/types": "5.29.0",
         "@sentry/webpack-plugin": "1.14.0",
         "@svgr/webpack": "5.5.0",
+        "@types/aws-sdk": "^2.7.0",
```
Comment (sr258) [inline package.json]: The S3 und Mongo stuff should be removed for Lumi.

**Comment by JPSchellenberg** [comment]
Hey,

I refactored the app so that the client is built by the create-react-app toolchain and merged the electron and server parts.
What do you think?

**Comment by sr258** [comment]
App doesn't work if I have this env var: NODE_ENV=development Without it it starts fine.

**Comment by JPSchellenberg** [comment]
Hmm, that's strange. Is PORT=8080?

**Comment by sr258** [comment]
Yes, the port is set to 8080

**Pull request merged**